requirements_dev.txt

```diff
@@ -1,3 +1,3 @@
-black==23.3.0
+black[d]==23.3.0
```

Comment:
`[d]` variant is to support black daemon, which the `BlackConnect` intellij plugin I use uses...